flow/util/distributed.py

```diff
@@ -205,6 +205,23 @@ def read(path):
 
     def read_sweep(this):
         return [*this['minmax'], this['step']]
+    
+    def apply_condition(config, data):
+        # TODO: tune.sample_from only supports 'random' search algorithm. 
+        # To make conditional paraeter for the other algorithms, different 
+        # algorithms should take different methods.
```

Comment:
What is still missing, i.e., why TODO?
paraeter -> parameter?
Prefer using docstrings (triple single/double quotes) is better for generating docs later.

Reply:
Thank you for your comment. I corrected the type and removed ''.

For basic ‘random’ search, the script succeed.
For the other search algorithms, it failed. It is because all other search algorithms do not support “tune.sample_from” command.
Hyperopt and Ax support conditional parameter, so instead of wrapping the input parameter config dictionary by ‘tune’ namespace, it seems possible to wrap it in each algorithm namespace.  E.g., tune.randint(min, max) → hp.randint(parameter_name, min, max)

So, I'll update later for Hyperopt and Ax algorithm.